runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -226,11 +225,7 @@ NNFW_STATUS nnfw_session::load_circle_from_buffer(uint8_t *buffer, size_t size)
     std::cerr << "Error during model loading : " << e.what() << std::endl;
     return NNFW_STATUS_ERROR;
   }
-
-  _tracing_ctx = std::make_unique<onert::util::TracingCtx>(_subgraphs.get());
-
-  _compiler = std::make_unique<onert::compiler::Compiler>(_subgraphs, _tracing_ctx.get());
-
+  _coptions = std::make_unique<onert::compiler::CompilerOptions>(*_subgraphs);
```

Comment:
`CompilerOption` has to be created here because `set_available_backends`, `set_config` and so on may change compiler default options before invoking `prepare()`.